Documentation/Contributors/BuildGuide/README.md

```diff
@@ -26,7 +26,7 @@
 ## Build the Code
 
 Prerequisites:
- * Install [Node.js](http://nodejs.org/) on your system.
+ * Install [Node.js](http://nodejs.org/) on your system.  Cesium requires Node 4.x or newer.
```

Comment:
Can we change this to `Building Cesium requires Node 4.x or newer.` otherwise I could see some new users getting confused (since Cesium itself doesn't even require node to work in a browser).